readme.md

```diff
@@ -62,16 +62,17 @@ body{
 All the steps for one way of doing the above have been written below, but in the wrong order. Put them in the correct order.
 
 ```
+$ cd project-repo
+$ git init project-repo
```

Comment:
`git init project-repo` actually creates a folder called `project-repo` with a `.git` folder inside it. So `git init...` would come first, then `cd...`.